ethsigner/core/src/test/java/tech/pegasys/ethsigner/core/requesthandler/sendtransaction/transaction/GoQuorumEthTransactionTest.java

```diff
@@ -44,7 +44,7 @@ public void setup() {
     params.receiver("0xd46e8dd67c5d32be8058bb8eb970870f07244567");
     params.gas("0x76c0");
     params.gasPrice("0x9184e72a000");
-    params.nonce("0x07");
+    params.nonce("0x7");
```

Comment:
is this related to a dependency upgrade?

Reply:
yes, web3j upgrade (either directly or transitively) has introduced this limitation i.e. the nonce format. It doesn't allow leading 0 in hex values anymore.